scala/lambdas/ingest-validate-generic-ingest-inputs/src/test/scala/uk/gov/nationalarchives/ingestvalidategenericingestinputs/MetadataJsonValueValidatorTest.scala

```diff
@@ -44,6 +44,9 @@ class MetadataJsonValueValidatorTest extends AnyFlatSpec with MockitoSugar with
     ("Files", Map(3 -> 0, 4 -> 0), "ArchiveFolder")
   )
 
+  private val casesWhereEntryShouldBeLabelledAsMetadataFile: TableFor2[Boolean, String] =
```

Comment:
This name is far too long. `entriesLabelledAsMetadataFile` perhaps or something similar

Reply:
Hmm, it doesn't quite describe what it is but I could name it `metadataFileCases`?

Reply:
Yeah ok :+1: